packages/sdk/src/types/json.ts

```diff
@@ -72,7 +72,8 @@ function reviveConcordiumTypes(value: unknown) {
 }
 
 /**
- * Acts as an inverse for {@linkcode jsonStringify}
+ * Acts as an inverse for {@linkcode jsonStringify}.
+ * @deprecated Use an `AccountTransactionHandler` instance instead.
```

Comment:
I guess this is not correct, as this can take any structure of concordium domain types (not just account transaction payloads).
```suggestion
 * @deprecated Manually convert the JSON structure instead. For account transactions, parse the output with something that handles deserializing`bigint`s, e.g. the `json-bigint` dependency, and use `AccountTransactionHandler.fromJSON`.
```